### PR TITLE
docs: add influxdb demo

### PR DESCRIPTION
**Pull request by glenngillen**
Adding in a reusable demo to compliment the [blog post about InfluxDB](https://www.ockam.io/blog/connect_private_influxdb).

### CODE REVIEW

**Comment by glenngillen** [comment]
Not sure how to tell it to skip over running the shell linting in the demo folders. The file it's erroring on is a copy of an InfluxDB managed script so not really something I want to be reformatting unnecessarily.

**Comment by etorreborre** [comment]
@glenngillen I guess you can try to tweak the `find` search [here](https://github.com/build-trust/ockam/blob/develop/.github/workflows/shell.yml#L57) so that it skips the `demo` folders.

File: demos/influxdb-telegraf/bin/setup [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        Yes ) break;;
+        No ) exit;;
+    esac
+done
+ockam reset
+ockam enroll
+
+PROJECT_NAME="default"
+result=$(ockam project information $PROJECT_NAME &>/dev/null)
+# if [ ! $? -eq 0 ]; then
```
Comment (etorreborre) [inline demos/influxdb-telegraf/bin/setup]: Let's remove the comments if we don't use that code

**Review by etorreborre — DISMISSED** [review]
Looks all good to me

**Review by etorreborre — APPROVED** [review]

**Pull request merged**